Core/src/main.cpp

```diff
@@ -142,9 +140,8 @@ int run_glfw() {
                     fixedUpdateBuffer -= PHYSICS_UPDATE_INTERVAL;
                 }
                 std::this_thread::sleep_for(std::chrono::microseconds(1));
-                audioManager->updateListenerPosition(cameraEnt->transform->getWorldPosition());
-                audioManager->updateSoundPosition(testCarEntity->transform->getWorldPosition());
-                audioManager->Update(0.0);
+                audioManager->updateListenerPosition(cameraEnt);
+                audioManager->Update(deltaTimeFloatSeconds);
```

Comment:
Should update the name physicsthread to physics and audio thread or something more descriptive for audio purposes.

Reply:
Will do.